docs/source/codependence/correlation_based_metrics.rst

```diff
@@ -159,6 +159,57 @@ Implementation
 
 .. autofunction:: squared_angular_distance
 
+Kullback-Leibler Distance
+=========================
+
+The Kullback-Leibler distance is a measure of distance between two probability densities, say p and q, which is defined as
+
+.. math::
+    K(p,q) = E_p \left[log\left(\frac{p}{q}\right)\right]
+
+Where :math:`E_p[.]` indicates the expectation value with respect to the probability density :math:`p`. Here we consider
+the Kullback-Leibler between multivariate Gaussian random variables (aka. Correlation matrices).
+
+Given two positive definite correlation matrices :math:`C_1` and :math:`C_2` assosiated with random variable :math:`X`, we can compute
+their probabilty density functions to :math:`P(C_1,X)` and :math:`P(C_1,X)` resulting in the following formula
+
+.. math::
+     K(P(C_1,X),P(C_2,X)) = E_{P(C_1,X)} \left[log\left(\frac{P(C_1,X)}{P(C_2,X)}\right)\right] = \frac{1}{2}\left[log\left(\frac{|C_1|}{|C_2|}\right)+tr(C^{-1}_2 C_1) -n) \right]
+
+where :math:`n` is the dimension of the space spanned by :math:`X`, and :math:`|C|` indicates the determinant of :math:`C`
+
+The Kullback-Leibler distance can be used to measure the stability of filtering/de-noising procedures with respect to statistical
+uncertainty.
+
+.. tip::
+    It is worth noting that the Kullback-Leibler distance takes naturally into account the statistical nature of correlation matrices
+    which is uncommon with other measures of distance between matrices such as Frobenius distance which is based on
+    the iso-morphism between the matrices space and the vectors space.
+
+Implementation
+##############
+
+.. autofunction:: kl_dist
+
+Norm Distance
+=============
```

Comment:
This is the wrong style header isn't it? 

Norm Distance should not be a H1 header since it is a subheading. So at best its a H2 heading.

Reply:
@PanPip can you please fix the headers in this file. I see that @skyliquid22 is following the convention in this file but its wrong.

Reply:
@Jackal08 will do.

Reply:
Done

Reply:
Thanks